Dockerfile

```diff
@@ -23,7 +44,9 @@ RUN mkdir -p /home/${USER}/.radicle/${USER}
 # Change the ownership of the working directory to seed
 RUN chown -R ${USER}:${GROUP} /home/${USER}
 
-COPY /creature-pigeon /home/seed/creature-pigeon
```

Comment:
I created this folder to be initialized as the Radicle repository to start off with. Why was it considered to be redundant?

Reply:
Oh my mistake I didn't get that.
I will put it back as it was.

Reply:
The problem is that it's not adding a real repo. 
Why not add the repo itself?

Reply:
Oh so my initial idea was to have a Radicle reposiory inside this deployed Radicle node with nothing but a README file. This repo is just for testing whether we can clone and make changes to it or not. 
You're right that we should add the repo itself and create other Radicle private repos inside of it later. I will make some adjustments to the initializer.sh file as there were some parts related to this repo! 
Also another thing is if we start creating Radicle private repos inside this, integrating the `creature-shoveler` into it is feasible. Am I right?

Reply:
Yeah either we add this repo itself or nothing and have peers push repos. Either public or private.

I like that we add a repo that way we can validate that it works but seeing the repo via the UI or fetching it via rad commands. 

Yes the idea would be to add private repos of course.

Please do rename this to osprey soon so that I can complete this work with DNS and static IP for fly to bind properly.

Reply:
I added back the current directory.